lib/express_validator.js

```diff
@@ -85,6 +85,13 @@ var expressValidator = function(options) {
   });
 
   ValidatorChain.prototype.notEmpty = function() {
+    if (this.value == null) {
```

Comment:
For undefined or null values, validator.js would coerce them to empty strings and perform validation on them as empty strings. This code changes that behavior by making null and undefined values throw an error instead. This would be a major change.

For this PR, I don't think major changes should be made so users can safely upgrade to this version and get rid of that annoying warning without having to modify their code. I'll discuss this further in a PR comment as well as my thoughts on how best to address undefined values for a later PR.